modules/modifying-an-existing-ingress-controller.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-an-existing-ingress-controller_{context}"]
+= Modifying an existing Ingress Controller
+
+As a cluster administrator, you can modify an existing Ingress Controller to manage the DNS record lifecycle.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* A DNS record is created and published.
+
+.Procedure
+
+. Modify the custom `IngressController` custom resource to set `dnsManagementPolicy`:
+
++
+[source,terminal]
+----
+oc -n openshift-ingress-operator patch ingresscontrollers/<name> --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"type":"LoadBalancerService","loadBalancer":{"dnsManagementPolicy":"Unmanaged", "scope":"<current-scope>"}}}}'
+----
+
+. Save the file to apply the changes.
++
+[source,terminal]
+----
+oc apply -f <name>.yaml <1>
+----
```

Comment:
There's no file in this section.

Reply:
From my understanding Step 1 , we modify an existing yaml file for the object `IngressController` and then save the file to apply the changes. 
for example ref  https://docs.openshift.com/container-platform/4.11/security/allowing-javascript-access-api-server.html

Reply:
The command on L23 (https://github.com/openshift/openshift-docs/pull/49752/files#diff-575bf2702aeabae30bfbe683d6276cedff8092f80209dcae8d7ed7b42959f83eR23) is run instead of editing a file.

Reply:
@Amrita42 `Step 2` should be deleted.

Reply:
Thanks @candita implemented.